### DISCUSSION TITLE
Does ImageGlass support resizing images?

### DISCUSSION THREAD

**Discussion by ghost** [d2phap/ImageGlass#790] [General] [answered]
### System information:
- ImageGlass version: 7.6
### Questions
Does ImageGlass support resizing images? "Image dimensions" opens an OS menu.
Do I have to use an external application for resizing?

**Comment by d2phap** [comment, marked as answer]
No it doesn't, ImageGlass mainly focuses on viewing functions.
However, you can use "Edit" feature to open in other apps.
